tests/rackups/timeout.ru

```diff
@@ -4,6 +4,10 @@ class App < Sinatra::Base
   set :environment, :production
   enable :dump_errors
 
+  get('/') do
```

Comment:
Conversely, there is some nice simplicity in just using one rackup throughout, but in that case I still think it might be nice to make it be something that is more explicit and hopefully obvious. So maybe we could have this be something like `get('no-timeout')` or something? That way it hopefully it easier to remember and see what is expected. What do you think?

Reply:
I didn't use basic because I wasn't sure I wanted to run 2 servers for these tests. I like the `no-timeout` suggestion.